hack/virtc/main.go

```diff
@@ -227,7 +228,14 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 		Memory: vmMemory,
 	}
 
+	id := context.String("pod-id")
```

Comment:
@jodh-intel `podID` should be a `UUID`, yes. I agree they should just be assigned by virtcontainers and not exported.

I think `containerID` should also be a `UUID`, not to be confused with a container name (That we don't have atm).

So overall we should stop exporting the `ID` field from both `ContainerConfig` and `PodConfig` and only generate valid `UUID`s for those. 
And we should add a `Name` exported field to both `ContainerConfig` and `PodConfig` that ican be defined by the caller or we'd generate a readable one. Here only `[a-ZA-Z0-9-_.]` would be acceptable. This can be done as a separate PR.